.github/workflows/ubi8-bats-pr.yaml

```diff
@@ -0,0 +1,37 @@
+name: ubi8-bats-pr
+on:
+  pull_request:
+    paths:
+      - utilities/ubi8-bats/**
+jobs:
+  build:
+    env:
+      context: utilities/ubi8-bats
+      image_name: ubi8-bats
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Check and verify version.json
+        id: check_version
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          echo $(jq -r '.version' ${context}/version.json)
+      - name: Build image
+        uses: docker/build-push-action@v1
+        with:
+          path: ${{ env.context }}
+          push: false
+          repository: ${{ env.image_name }}
+          tags: ${{ steps.check_version.outputs.IMAGE_TAGS }}
+      - name: Test image
+        run: |
+          echo "Running: docker run --entrypoint bash ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} -c 'bats --version'"
```

Comment:
@pabrahamsson ; can we not use bats to do all these tests?

Reply:
We sure can, I've updated the PR